commander/tsconfig.json

```diff
@@ -2,7 +2,6 @@
 	"extends": "../tsconfig.json",
 	"compilerOptions": {
 		"composite": true,
-		"declaration": true,
```

Comment:
Why we had to remove this option?

Reply:
It is defined in `"extends": "../tsconfig.json",` now, as framework was not completely typescript, it was defined in each one